data/mods/Magiclysm/items/enchanted_cloaks.json

```diff
@@ -4,7 +4,7 @@
     "type": "enchantment",
     "has": "WORN",
     "condition": "ALWAYS",
-    "mutations": [ "GILLS", "MEMBRANE", "AMPHIBIAN", "WILDSHAPE:FISH" ]
+    "mutations": [ "GILLS", "MEMBRANE", "AMPHIBIAN", "WAYFARER", "WILDSHAPE:FISH" ]
```

Comment:
This feels very hacky, I suppose it's meant to represent the physical inability to control a vehicle, it's weird to use wayfarer for that.

Reply:
Hi, do you have a suggestion for an alternative? There is precedence for this in that Bear already had this trait. The only other alternative I can think of is to simply remove the trait and not include the restriction.

Reply:
I guess it's fine for now, and can be revisited once limb infrastructure gets more advanced